src/components/RecipeEdit.vue

```diff
@@ -89,7 +93,18 @@ export default {
                 { key: 'sodiumContent',label: t('cookbook', 'Sodium content'), placeholder: t('cookbook', 'E.g.: 2 g (amount & unit)') },
                 { key: 'sugarContent', label: t('cookbook', 'Sugar content'), placeholder: t('cookbook', 'E.g.: 2 g (amount & unit)') },
                 { key: 'transFatContent', label: t('cookbook', 'Trans-fat content'), placeholder: t('cookbook', 'E.g.: 2 g (amount & unit)') },
-                { key: 'unsaturatedFatContent', label: t('cookbook', 'Unsaturated-fat content'), placeholder: t('cookbook', 'E.g.: 2 g (amount & unit)') }]
+                { key: 'unsaturatedFatContent', label: t('cookbook', 'Unsaturated-fat content'), placeholder: t('cookbook', 'E.g.: 2 g (amount & unit)') }],
+            referencesPopupFocused: false,
+            popupContext: undefined,
+            loadingRecipeReferences: true
+        }
+    },
+    computed: {
+        allrecipeOptions() {
+            return this.allRecipes.map(r => {
+                    return {recipe_id: r.recipe_id, name: r.recipe_id + ": " + r.name}
```

Comment:
Maybe renaming to `title` instead of `name` might be favorable to have an open to extend later.

Reply:
sounds sensible